tagger/app.py

```diff
@@ -0,0 +1,69 @@
+import logging
+
+import boto3
+
+
+LOG = logging.getLogger(__name__)
+LOG.setLevel(logging.DEBUG)
+
+
+tag_client = boto3.client('resourcegroupstaggingapi', 'us-east-1')
```

Comment:
is the region required?  if so i would parameterize it and pass it in from the cloudformation template.